packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2920,6 +2920,12 @@ export class ContainerRuntime
 	 * data store or an attachment blob.
 	 */
 	public async getGCNodePackagePath(nodePath: string): Promise<readonly string[] | undefined> {
+		// GC uses "/" when adding "root" references, e.g. for Aliasing or as part of Tombstone Auto-Recovery.
+		// These have no package path so return a special value.
+		if (nodePath === "/") {
+			return ["<GCROOT>"];
```

Comment:
Why the <> and not just "GCROOT" or "ROOT"?

Reply:
I dunno just to highlight that it's not an actual package path (as obvious as it'll be). Which do you prefer?

Reply:
I was just curious if this format meant anything specific :-). We use "_blobs" for attachment blobs. I don't have a preference on which one to use but we should use similar formatting for both of them.